lib/style.css

```diff
@@ -101,6 +115,8 @@ body {
 }
 
 .material .song-row .song-indicator {
+  background-image: url('ani_equalizer_white_x2.gif');
+  background-size: 28px 28px;
```

Comment:
Any reason for `background-size` here? It seems like the existing indicator should handle that if it is using `background-image`.